Mathlib/Combinatorics/Quiver/Basic.lean

```diff
@@ -145,15 +145,15 @@ namespace Quiver
 
 /-- `Vᵒᵖ` reverses the direction of all arrows of `V`. -/
 instance opposite {V} [Quiver V] : Quiver Vᵒᵖ :=
-  ⟨fun a b => (unop b ⟶ unop a)ᵒᵖ⟩
+  ⟨fun ⟨a⟩ ⟨b⟩ => (b ⟶  a)ᵒᵖ⟩
```

Comment:
Why do we need this change?

Reply:
I don't know if it's necessary but see https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/optimal.20instance.20design.20for.20structure.20wrappers

Reply:
Well an example immediately popped up: removing `Quiver.Hom.unop_op` and `Quiver.Hom.op_unop` breaks `aesop` automation in `id_comp` and `comp_id` for `CategoryTheory.Category.opposite`. However this works fine
```lean
instance Category.opposite : Category.{v₁} Cᵒᵖ where
  comp := fun ⟨f⟩ ⟨g⟩ => ⟨g ≫ f⟩
  id := fun ⟨X⟩ => ⟨𝟙 X⟩
```